### PR TITLE
Nginx - update to version 1.20.1

### PR DESCRIPTION
**Pull request by EugeneKorolevich**

### CODE REVIEW

File: 1/README.md [hunk 1]
```diff
@@ -1,406 +0,0 @@
-# <a name="about"></a>About
```
Comment (ovk6) [inline 1/README.md]: Revert readmes. Should not be deleted

**Review by ovk6 — CHANGES_REQUESTED** [review]

File: 1/README.md (hunk 1 above)
Comment (EugeneKorolevich) [reply 1/README.md]: done

File: versions.yaml [hunk 2]
```diff
@@ -29,6 +29,28 @@ _variables:
     gpg: '08a0a054aecedec4aa2772569325a346403d2c1fba6f88739553416c2f4f77e6'
 
 versions:
+  - dir: '1/debian9/1.18'
```
Comment (armandomiani) [inline versions.yaml]: As discussed, the latest stable nginx is 1.20.1

**Review by ovk6 — APPROVED** [review]

File: versions.yaml (hunk 2 above)
Comment (EugeneKorolevich) [reply versions.yaml]: updated to 1.20.1 with "buster" repo on Debian10

**Pull request merged**